idunn/blocks/website.py

```diff
@@ -1,12 +1,21 @@
 from .base import BaseBlock
 
-from typing import ClassVar
+from typing import ClassVar, Union, Optional
+
+from pydantic import BaseModel
+
+
+class WebsiteUrl(BaseModel):
+    url: str
+    redirect: bool = False
+    hash: Optional[str]
+    label: Optional[str]
 
 
 class WebSiteBlock(BaseBlock):
     BLOCK_TYPE: ClassVar = "website"
 
-    url: str
+    url: Union[str, WebsiteUrl]
```

Comment:
I think we should avoid exposing this complexity to the client.   
As you suggested, I would prefer to set a public base URL in Idunn settings, and use it directly to provide a redirected `url` when needed.
Then, the optional `label` could be added in the existing block (and the block format would not change).

Reply:
Ok, I went for the solution that required no modification in our configurations by default but I wasn't sure at all, it will be easy to revert :+1: